app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlAdapter.kt

```diff
@@ -341,7 +341,13 @@ class SessionControlAdapter(
 
     override fun onViewRecycled(holder: RecyclerView.ViewHolder) {
         when (holder) {
-            is PocketStoriesViewHolder -> holder.composeView.disposeComposition()
```

Comment:
I was seeing as much as 300ms needed for `composeView.onAttachedToWindow` everytime the holder was scrolled off an on screen again.
I don't think we need the composable destroyed and keeping it around definitely means a smoother UX.